src/database/queries/baseFields/updateByShortCode.sql

```diff
@@ -1,9 +1,8 @@
 UPDATE base_fields SET
```

Comment:
This file name should now be `createOrUpdateOne` and add in the upsert logic (borrowing from other upserts, such as the `UserGroupFunderPermission` operations)